econml/tests/test_statsmodels.py

```diff
@@ -0,0 +1,916 @@
+                                    assert np.all(np.abs(coef[i, j, 2:]) < precision)
+                                    assert np.all((lower[i, j, 2:] <= precision_int) &
+                                                  (upper[i, j, 2:] >= -precision_int)),\
+                                        (np.max(lower[i, j, 2:]),
+                                         np.min(upper[i, j, 2:]))
+                            XT = np.hstack([X, T])
+                            (X1, X2, y1, y2,
+                             X_final_first, X_final_sec, y_sum_first, y_sum_sec, n_sum_first, n_sum_sec,
+                             var_first, var_sec) = _summarize(XT, y)
+                            X = np.vstack([X1, X2])
+                            y = np.concatenate((y1, y2))
+                            X_final = np.vstack([X_final_first, X_final_sec])
+                            y_sum = np.concatenate((y_sum_first, y_sum_sec))
+                            n_sum = np.concatenate((n_sum_first, n_sum_sec))
+                            var_sum = np.concatenate((var_first, var_sec))
+                            first_half_sum = len(y_sum_first)
+
+                            class SplitterSum:
+                                def __init__(self):
+                                    return
+
+                                def split(self, X, T):
+                                    return [(np.arange(0, first_half_sum), np.arange(first_half_sum, X.shape[0])),
+                                            (np.arange(first_half_sum, X.shape[0]), np.arange(0, first_half_sum))]
+                            est = LinearDMLCateEstimator(
+                                model_y=LinearRegression(),
+                                model_t=LinearRegression(),
+                                n_splits=SplitterSum(),
+                                linear_first_stages=False,
+                                discrete_treatment=False).fit(y_sum,
+                                                              X_final[:, d:],
+                                                              X_final[:, :d_x],
+                                                              X_final[:, d_x:d],
+                                                              sample_weight=n_sum,
+                                                              sample_var=var_sum,
+                                                              inference=StatsModelsInference(cov_type='nonrobust'))
+                            coef = est.coef_.reshape(p, q, d_x + 1)
+                            lower, upper = est.coef__interval(alpha=.001)
+                            lower = lower.reshape(p, q, d_x + 1)
+                            upper = upper.reshape(p, q, d_x + 1)
+                            for i in range(p):
+                                for j in range(q):
+                                    assert (np.abs(coef[i, j, 0] - 10 * i - j) <
```

Comment:
The parenthesis here needs to only wrap the comparison not the whole thing. my mistake

Reply:
Fixed in next push